sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/hybridsearch/HybridSearchQueryResult.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.query.hybridsearch;
+
+import com.azure.cosmos.implementation.Document;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HybridSearchQueryResult<T> extends Document {
+    public String rid;
+    public List<Double> componentScores;
+    public Document payload;
+    public Double score;
+
+    public HybridSearchQueryResult(String jsonString) {
+        super(jsonString);
+    }
+
+    public String getRid() {
```

Comment:
getRid/setRid should use/override getResourceId setResourceId instead (it is already defined in the super (Resource))